client/tokenized-payment-request/cart-api.js

```diff
@@ -0,0 +1,131 @@
+/* global jQuery */
+
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { applyFilters } from '@wordpress/hooks';
+
+export default class PaymentRequestCartApi {
+	// Used on product pages to interact with an anonymous cart.
+	// This anonymous cart is separate from the customer's cart, which might contain additional products.
+	// This functionality is also useful to calculate product/shipping pricing (and shipping needs)
+	// for compatibility scenarios with other plugins (like WC Bookings, Product Add-Ons, WC Deposits, etc.).
+	cartRequestHeaders = {};
+
+	async placeOrder( paymentData ) {
+		return await apiFetch( {
+			method: 'POST',
+			path: '/wc/store/v1/checkout',
+			credentials: 'omit',
+			headers: {
+				'X-WC-Payments-prb-request': true,
```

Comment:
```suggestion
				'X-WooPayments-Express-Payment-Request': true,
```
This is a minor quibble, but perhaps we should reconsider the name of this header. This implementation currently specifically supports Stripe's PRB integration, but the PRB itself will soon be deprecated, so this new feature will hopefully greatly outlive the soon-to-be forgotten PRB. I'd maybe suggest some new name like `Express-Checkout-Request` or `Express-Payment-Request` just to disconnect this component semantically from the PRB itself. (Also, this is an even more minor quibble, but WC Payments is no more, we WooPayments now.)

Reply:
I like the thought you put into this, I fully agree with reconsidering the `prb` prefix - changed to `Express-Payment` 👍 
In regards to WCPayments & WooPayments - there are still a bunch of items that refer the old name, so that's why I went with that 😬 but I'm totally open to having `WooPayments` 👍

Reply:
> In regards to WCPayments & WooPayments - there are still a bunch of items that refer the old name

I think most of our enqueued global variables will be prefixed with `wcpay...`, `WCPayAPI` still exists, and of course our gateway slug is still `woocommerce_payments`. These are remnants of antique code, but I don't think there's really a directive here and even if we future-proof the header name here to use the current plugin name rather than the old one, ultimately we'll be the only ones using this header at all, as I don't believe its intention is to be user-facing. This one's up to you.